### PR TITLE
[Filter] Update filter manual @open sesame 12/04 09:33 

### PR DESCRIPTION
**Pull request by gichan-jang**
Since new features of the filter are added, update the manual.

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_filter/README.md [hunk 1]
```diff
@@ -1,72 +1,83 @@
...
+The input/output stream data type is either ```other/tensor``` or ```other/tensors```.  
+  - With ```other/tensor```, you may push (or pull) a single tensor for an instance of inference for the given model.  
+  - With ```other/tensors```, you may push (or pull) multiple tensors for an instance of inference for the given model.  
+
+For example, if the output of a neural network model has two distinctive output tensors, "Bounding boxes, uint32[100][4]" and "Labels, uint32[100]", you may use ```other/tensors``` as the source pad capability to have both tensors in a single frame of the source pad.  
+Note that ```tensor_filter``` supports one always source pad and one always sink pad.
+
+## Example launch line (tensorflow)
+```
+... ! other/tensor(s) ! tensor_filter framework=tensorflow model=${PATH_TO_MODEL} input=784:1 inputtype=float32 inputname=input output=10:1 outputtype=float32 outputname=softmax ! tensor_sink
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_filter/README.md:13]: How about splitting this launch line to see the whole pipeline at a glance?

Comment (gichan-jang) [reply gst/nnstreamer/tensor_filter/README.md]: Thanks for the suggestion. Updated example launch line.

**Review by dongju-chae — APPROVED** [review]

**Review by myungjoo — APPROVED** [review]
